exiftool_test.go

```diff
@@ -614,3 +615,26 @@ func copyFile(src, dest string) (err error) {
 	}
 	return nil
 }
+
+func TestFailOnDirectoryInput(t *testing.T) {
+	//  https://github.com/barasher/go-exiftool/issues/52
+	e, err := NewExiftool()
+	require.Nil(t, err)
+	defer e.Close()
+
+	// create test directory with about 200MB worth of photos
+	// 10,000 of the included jpg or 6 30MB CR2's both repro
+	tmpPath := t.TempDir()
```

Comment:
This test is unnecessarily complex for testing the directory handling behavior (it can just pass in the testdata directory as is, or create an empty folder). Also, I believe you still have to clean up the temporary folder otherwise you are leaving unnecessary files on the host machine.

Reply:
[`t.tempDir()`](https://pkg.go.dev/testing#T.TempDir) creates an empty folder that is automatically removed once test has run.
I agree for the test complexity.

Reply:
my bad, I was thinking of `ioutil.TempDir()`. I learned something new, thanks :)

It looks like that was introduced in version 1.15. How does that work with this package being set to version 1.13? I guess as long as the runtime is 1.15 or higher it must not fail based on the go mod version.